src/pretix/presale/templates/pretixpresale/event/checkout_payment.html

```diff
@@ -62,58 +62,52 @@
                                         <strong class="pull-right flip">{% if p.fee < 0 %}-{% else %}+{% endif %} {{ p.fee|money:event.currency|cut:"-" }}</strong>
                                     {% endif %}
                                     <input type="radio" name="payment" value="{{ p.provider.identifier }}"
                                            title="{{ p.provider.public_name }}"
-                                           data-parent="#payment_accordion"
                                            {% if selected == p.provider.identifier %}checked="checked"{% endif %}
-                                           id="input_payment_{{ p.provider.identifier }}"
-                                           aria-describedby="payment_{{ p.provider.identifier }}"
-                                           data-toggle="radiocollapse" data-target="#payment_{{ p.provider.identifier }}"
+                                           aria-controls="payment_{{ p.provider.identifier }}"
                                            data-wallets="{{ p.provider.walletqueries|join:"|" }}" />
-                                    <label for="input_payment_{{ p.provider.identifier }}"><strong>{{ p.provider.public_name }}</strong></label>
-                                </p>
-                            </div>
-                        </div>
-                        <div id="payment_{{ p.provider.identifier }}"
-                             class="panel-collapse collapsed {% if selected == p.provider.identifier %}in{% endif %}">
-                            <div class="panel-body form-horizontal">
-                                {% if request.event.testmode %}
-                                    {% if p.provider.test_mode_message %}
-                                        <div class="alert alert-info">
-                                            <p>{{ p.provider.test_mode_message }}</p>
-                                        </div>
-                                        {% if not request.sales_channel.type_instance.testmode_supported %}
-                                            <div class="alert alert-danger">
-                                                <p>
-                                                {% trans "This sales channel does not provide support for test mode." %}
-                                                <strong>
-                                                    {% trans "If you continue, you might pay an actual order with non-existing money!" %}
-                                                </strong>
-                                                </p>
-                                            </div>
-                                        {% endif %}
-                                    {% else %}
-                                        <div class="alert alert-warning">
+                                    <strong class="provider-name">{{ p.provider.public_name }}</strong>
```

Comment:
Should we name it something more general, so it could be used somewhere else as well? Maybe at some point we like to include it in a SSO-adapter that does some Ajax-loading and wants to a loading indicator? What about something like `.accordion-panel-label`? Of course the js needs to be adatped then as well.

```suggestion
                                    <strong class="accordion-panel-label">{{ p.provider.public_name }}</strong>
```

Reply:
```suggestion
                                    <strong class="accordion-label-text">{{ p.provider.public_name }}</strong>
```